src/test/java/seedu/address/model/food/FridgeTest.java

```diff
@@ -0,0 +1,118 @@
+package seedu.address.model.food;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.food.exceptions.FoodNotFoundException;
+
+class FridgeTest {
+    private static final Food CHIMKEN = new Food("Chimken", 1, 2, 3);
+    private static final Food RAMEN = new Food("Ramen", 2, 3, 4);
+
+    private final Fridge fridge = new Fridge();
```

Comment:
Might be better to put it into a method rather than just declaring

Can use @Before to initialise the fridge to reduce the complexity of layering the different test cases on top of one another (This might help if we decide to change/add more test cases in the future)
    @Before
    public void beforeEachTestMethod() {
        System.out.println("Invoked before each test method");
    }

Reply:
Got it!